plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -284,6 +312,19 @@ fn main() {
         }
     };
 
+    if let PluginOp::List {
+        ref mut name,
+        ref mut maintainer,
+    } = apt.operation
+    {
+        if name.is_none() && maintainer.is_none() {
+            if let Some(config) = get_config() {
+                *name = config.apt.name;
+                *maintainer = config.apt.maintainer;
+            }
+        }
```

Comment:
Here the configuration file is used if nothing is provided on the command line. I would do something slightly different, taking the values provided by the configuration file unless explicitly provided on the command line.